carbon-design-system/carbon
# fix: filterable multiselect readonly state implemented

Closes #17587 

read only state for filterable multiselect is implemented

#### Changelog

**New**

- added new `readOnly` prop
- new `readOnlyEventHandlers` method to handle events during the `readOnly` state.

**Changed**

- changes related to readOnly state


#### Testing / Reviewing

- Go to the 'Filterable' story in the Multiselect component in the deploy preview
- Turn on the `readOnly` prop and see if it works as expected.
